nautobot/extras/tests/test_jobs.py

```diff
@@ -33,19 +31,11 @@ def get_job_class_and_model(module, name):
 
 
 def create_job_result_and_run_job(module, name, *, data=None, commit=True, request=None):
-    """Test helper function to call get_job_class_and_model() then create a JobResult and call run_job()."""
+    """Test helper function to call get_job_class_and_model() then and call run_job_for_testing()."""
     if data is None:
         data = {}
     job_class, job_model = get_job_class_and_model(module, name)
-    job_content_type = ContentType.objects.get(app_label="extras", model="job")
-    job_result = JobResult.objects.create(
-        name=job_model.class_path,
-        obj_type=job_content_type,
-        job_model=job_model,
-        user=None,
-        job_id=uuid.uuid4(),
-    )
-    run_job(data=data, request=request, commit=commit, job_result_pk=job_result.pk)
```

Comment:
Would it be acceptable to leave this here and duplicate the test case so we can ensure backwards compatibility?

Reply:
Spoke with @Kircheneer. Just want to get a test case that calls `run_job` directly to ensure we don't break anything for anyone who still does/needs to write tests that way. Also ensuring we test `run_job` directly itself.